core-java-modules/core-java-concurrency-collections-2/src/main/java/com/baeldung/concurrent/lock/ConcurrentAccessBenchmark.java

```diff
@@ -0,0 +1,49 @@
+package com.baeldung.concurrent.lock;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+
+@State(Scope.Thread)
+@Fork(value = 1)
+@Warmup(iterations = 0)
+public class ConcurrentAccessBenchmark {
+    ConcurrentAccessExperiment accessMyMap;
```

Comment:
I think this would be simplified by having both instances of your test instantiated from the beginning:

```suggestion
    SingleLock single = new SingleLock();
    StripedLock striped = new StripedLock(BUCKETS);
```

Reply:
Now both are inited on test call